packages/flutter_tools/lib/src/commands/build_ios_framework.dart

```diff
@@ -232,6 +232,7 @@ class BuildIOSFrameworkCommand extends BuildSubCommand {
   void produceFlutterPodspec(BuildMode mode, Directory modeDirectory, { bool force = false }) {
     final Status status = globals.logger.startProgress(' ├─Creating Flutter.podspec...', timeout: timeoutConfiguration.fastOperation);
     try {
+      _flutterVersion.fetchTagsAndUpdate();
```

Comment:
Is there anywhere we request gitTagVersion where we wouldn't want to fetch the tags first?

Reply:
Yes.  We get the flutter version for basically every command, which does this work.  If we do this for every anayzer or build invocation it is a measurable performance hit.